### ISSUE TITLE
best way to handle 'no change' between recipe runs?

### ISSUE THREAD

**Issue by osterbit** [qri-recipes/overview#4]
- qri will intentionally fail at making a commit when a dataset has no changes
- in general this is desired, however if I am running a recipe on a regular interval and would like to know that the value of a variable is the same at both times I ran the recipe, I think I would want a log of some sort so that I can distinguish between 'no change' and 'recipe didn't run'

@b5 -- thoughts?

**Comment by b5** [comment]
absolutely. thinking in terms of "exit status" I think the standard approach to a qri recipe should be to print "no changes detected" and exit "status 0" (no error). We should update documentation in this repo to reflect this

**Comment by osterbit** [comment]
so would we want to document that in a log dataset maybe as "me/{dataset_name}_log" to record the recipe running and add that name to the metadata of the actual dataset? (thinking of the history querying use-case where maybe I want to get the daily availability of a product and have my summary show a timestamped observation even if no availability changed that day)
